WordPress/Resources/release_notes.txt

```diff
@@ -1,3 +1,3 @@
-We don’t know about you, but we’re feeling 22. In honor of the occasion (and because you’ve been asking for it), you can now transform most types of blocks into other block types, like quotes, columns, and groups. Nice.
-
-You’ll also see a few small visual changes to the My Site dashboard and the in-app survey.
+* [**] [internal] Refactor updating account related Core Data operations, which ususally happens during log in and out of the app. [#20394]
+* [***] [internal] Refactor uploading photos (from the device photo, the Free Photo library, and other sources) to the WordPress Media Library. Affected areas are where you can choose a photo and upload, including the "Media" screen, adding images to a post, updating site icon, etc. [#20322]
+* [**] Warns user about sites with only individual plugins not supporting core app features and offers the option to switch to the Jetpack app. [#20408]
```

Comment:
The raw notes include an entry that is not marked as "Jetpack-only" but that applies only to Jetpack:

https://github.com/wordpress-mobile/WordPress-iOS/blob/f316bf49a4515528ed7e1b26414463183c366696/RELEASE-NOTES.txt#L6

I decided to leave it out.

Reply:
Good call @mokagio! I reviewed this PR but didn't realize it was missing the `Jetpack-only` prefix. Thanks for correcting it.